app/views/shared/views/_alerts_list.html.haml

```diff
@@ -0,0 +1,110 @@
+.col-md-12.list-view-container.list-view-compact
+  .list-view-pf-view{"pf-list-view" => "",
+       "config" => "vm.listConfig",
+       "items" => "vm.alertsList",
+       "menu-actions" => "vm.menuActions",
+       "update-menu-action-for-item-fn" => "vm.updateMenuActionForItemFn",
+       "custom-scope" => "vm"}
+    .list-view-pf-left
+      %span{:class=> "list-view-severity {{item.severityInfo.severityIconClass}}",
+            "tooltip" => "{{item.severityInfo.title}}",
+            "tooltip-placement" => "bottom",
+            "tooltip-popup-delay" => "1000"}
+    .list-view-pf-body
+      .row
+        .col-md-3.col-sm-4.col-xs-6
+          %div
+            %a{"href" => "#", "ng-click" => "customScope.showHostPage(item, $event)"}
+              %span.no-wrap{"tooltip" => "{{item.hostName}}",
+                            "tooltip-placement" => "bottom",
+                            "tooltip-popup-delay" => "1000"}
+                %img.list-view-type-img{"ng-src" => "/assets/100/container_node.png"}
```

Comment:
Same problem here, this needs to be `:src => image_path("100/container_node.png")`